src/transformState.js

```diff
@@ -4,8 +4,35 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const obj in actions) {
```

Comment:
If you iterate through an array of `actions`, the element should be named `action`, right?